github-labeling-bot/.gitignore

```diff
@@ -1 +1,3 @@
 bot/config/config.yml
+.bundle
```

Comment:
how is this related? :confused:

Reply:
It is not but do you really expect me to create a dedicated PR for this? It is in a dedicated commit...